### PR TITLE
Support building multiple versions of bazelbuild image d190723

### PR DESCRIPTION
**Pull request by ixdy**
This basically combines techniques from the planter Makefile/Dockerfile and the kubekins Makefile/Dockerfile to give us three similar images that only differ by the bazel version included. Included here are 0.5.2 (for kubernetes 1.6), 0.5.4 (for 1.7 and 1.8), and 0.6.0 (for master, soon).

I've also:
* moved the source location from `pull_kubernetes_bazel` to `bazelbuild`, to match the image's name
* simplified the apt-get dependency list
* bumped gcloud to 173.0.0

`make build` works. I haven't done any additional testing yet.

/assign @BenTheElder @krzyzacy

### CODE REVIEW

File: images/bazelbuild/Dockerfile [hunk 1]
```diff
@@ -16,26 +16,21 @@
...
-    rm "bazel_${BAZEL_VERSION}-linux-x86_64.deb"
+    python-pip \
+    unzip \
+    wget \
+    zip \
+    zlib1g-dev \
+    && apt-get clean
 
-ENV GCLOUD_VERSION 163.0.0
+ENV GCLOUD_VERSION 173.0.0
```
Comment (BenTheElder) [inline images/bazelbuild/Dockerfile]: @krzyzacy is this good to go? :-)

**Comment by BenTheElder** [comment]
/lgtm
:fire:

**Comment by ixdy** [comment]
I've pushed these to gcr.io. test away!

**Comment by krzyzacy** [comment]
do we want to combine bazelbuild image with kubekins?

**Comment by ixdy** [comment]
maybe. I wonder what dependencies we're missing besides kubetest.

**Pull request merged**